interface/backend/static/views.py

```diff
@@ -0,0 +1,7 @@
+from django.shortcuts import render
+
+
+def home(request):
+    current_hash = open('/HEAD', 'r').readlines()[-1].split(' ')[1]
```

Comment:
So, there is "prior art" within Django for this kind of lookup - I think it's called ``get_version`` or similar. Basically, it needs to a) cache and b) be in some utility method.

(Also, I prefer ``with`` statements; we're relying on garbage collection here to close the file descriptor)

Reply:
Another approach would be to do this once per instance in ``settings.py`` and assign to VERSION = "whatever".

Reply:
It also requires some kind of fallback- lots of things clean ".git" etc. etc so falling back to "(unknown)" on any error seems sensible.

Reply:
Thanks for your feedback @lamby ! I think I changed everything accordingly. Though, when creating a new commit or switching branches, the version number is not updated. One is now required to restart the docker container but I think that shouldn't be such a big problem?